config/routes.rb

```diff
@@ -6,6 +6,7 @@
   resources :follow_requests
   resources :comments
   resources :photos
+  get "/:username" => "users#show", as: :user
```

Comment:
Make sure to include devise for users, I know you have it already, but it is meant to be inside this PR.